### PR TITLE
Fix Bug 1253473 - A change is needed to https://www.mozilla.org/en-US/foundation/licensing/website-content/

### PR DESCRIPTION
**Pull request by kyoshino**
This page isn't localized.

### CODE REVIEW

File: bedrock/foundation/templates/foundation/licensing/website-content.html [hunk 1]
```diff
@@ -38,8 +38,8 @@ <h1 class="page-title">{{ _('Mozilla.org Site Licensing Policies') }}</h1>
   <p>{{ _('If you have any questions about complying with our licensing terms for this collection, you should email:') }}</p>
 
   <ul>
-    <li><a href="mailto:trademarks@mozilla.com">trademarks@mozilla.com</a> {{ _('for Firefox or Thunderbird trademark and logo questions; or') }}
-    <li><a href="mailto:trademarks@mozilla.org">trademarks@mozilla.org</a> {{ _('for other trademark or logo questions; or') }} </li>
+    <li><a href="mailto:trademarks@mozilla.com">trademarks@mozilla.com</a> {{ _('for general trademark questions; or') }}
+    <li><a href="mailto:trademark-permissions@mozilla.com">trademark-permissions@mozilla.com</a> {{ _('for permission to use Mozilla's trademarks or logos; or') }} </li>
```
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/licensing/website-content.html]: Need a curly `’` here

**Comment by alexgibson** [comment]
Thanks @kyoshino r+wc :bow:

**Comment by kyoshino** [comment]
Ouch, will fix it :blush:

**Comment by kyoshino** [comment]
Fixed the l10n formatting as well!

**Comment by alexgibson** [comment]
Nice, thanks @kyoshino! r+

**Pull request merged**